workloads/network-perf/smoke-crd.yaml

```diff
@@ -1,9 +1,8 @@
 ---
-
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: uperf-benchmark-${WORKLOAD}-network-${pairs}
+  name: uperf-${WORKLOAD}-network-${PAIRS}
```

Comment:
Really a minor thought but shouldn't this be something like ```uperf-${WORKLOAD}-pairs-${PAIRS}``` ?
The word network in there is kind of redundant with uperf and at a glance the last number doesn't really say anything as it is today.

Reply:
fixed!